mart/attack/adversary.py

```diff
@@ -132,8 +133,8 @@ def done(self) -> bool:
 
         return False
 
-    def on_run_start(self, adversary, input, target, model, **kwargs):
-        super().on_run_start(adversary, input, target, model, **kwargs)
+    def on_run_start(self, **kwargs):
+        super().on_run_start(**kwargs)
```

Comment:
I would revert this change. Having explicit arguments is good here.

Reply:
Fixed.